src/Orleans.Core/Statistics/FloatValueStatistic.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Orleans.Runtime
 {
-    internal class FloatValueStatistic : ICounter<float>
```

Comment:
Should these be in Orleans.Runtime.Internal?  I don't think we want public using these surfaces as, imo, they are subject to change/removal.  If we at least hide them behind an "internal" namespace, then that should mostly hide them and indicate that we're not supporting them as part of our public surface.

Reply:
Above comment applies to all ICounters